client/src/Pages/BuyAndSellPage/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
+import Plot from 'react-plotly.js';
+import { DateTime } from "luxon";
+// import { useHistory } from "react-router-history"
+/*
+TODO:
+[] Figure out cors
+[] 
+*/
+
+
+export default () => {
+
+    const companyID = "uber"
+
+    const [isLoading, setIsLoading] = useState(true)
+    const [data, setData] = useState()
+    const [companyDetails, setCompanyDetails] = useState()
+
+    const [value, setValue] = useState(0);
+
+    /**
+     * 
+     * @param {int} value Number of stocks in transaction
+     * @param {boolean} buy Is it a buy or a sell transaction
+     */
+    const onSubmitHandler = async (value, buy) => {
```

Comment:
Looks great. Might want to consider putting this func outside the functional component. Just cuz javascript creates a new func instance on every draw. It's not really a problem, but also a really easy fix

Reply:
Done. I did not realize this